SubtitlesAPI/README.md

```diff
@@ -0,0 +1,59 @@
+# SubtitlesAPI
+
+A centralized API for fetching localized subtitles.
+
+## Feedback
+
+Got any feedback or found bugs? Please do post it in [the Github Issues page](https://github.com/JanGuillermo/LethalCompany/issues)!
+
+## Usage
+
+If you are creating a mod and you want to add subtitles for any custom sounds, it's pretty simple! Add the mod .dll file to your project references, and just add the following code:
+
+```cs
+using static SubtitlesAPI.SubtitlesAPI;
+
+try
+{
+  Localization.AddTranslation("CustomSound", "Custom subtitle");
+}
+catch
+{
+  Debug.Log("SubtitlesAPI Mod not Found");
+}
+```
+
+No need to even require players to use this mod!
+
+There is one method with an overflow that you can use: <br>`SubtitlesAPI.LocalizationTable.AddTranslation(string soundFileName, string subtitleText);` and `SubtitlesAPI.LocalizationTable.AddSound(Dictionary<string, string>);`<br>Where `Dictionary<string, string>` is a dictionary with `Key: soundFileName`, `Value: subtitleText`.
+
+The method will return a boolean, or list of booleans for the dictionary. The result will be true if the mod was able to add the subtitle, and false if not - which is likely due to the translation for the soundFileName already being created. Maybe another mod has the same sound file name?
+
+## Contributing
+
+Do you want to add or change any subtitles? Do you want to create a new locale for another language? Do you want to add more functionality to this mod? Please do! Create a fork of this repo, make additions or changes there, and submit a pull request.
```

Comment:
```suggestion
Do you want to add or change any subtitles? Do you want to create a new locale for another language? Please do! Create a fork of this repo, make additions or changes there, and submit a pull request.
```
Remove bit about adding more functionality - once this is complete, I doubt that adding more functionality will be necessary.